### DISCUSSION TITLE
Permalink problem

### DISCUSSION THREAD

**Discussion by Njthon** [11ty/eleventy#3603] [Q&A] [answered]
Hello all!

I recently discover Obsidian and the amazing plugin [Digital Garden Obsidian Plugin 1](https://github.com/oleeskild/digitalgarden).
I tried to put my first note online and it worked.
I’m having problems in publishing more content. Basically I got the following error: [obsidian permalink - Pastebin.com 1](https://pastebin.com/1kqFQywF) .

It is an error with permalinks, apparently it’s creating the same permalink for different notes…I guess. I am working locally with the repository, adding some HTML and, theoretically JS and CSS.

Edit: I deleted and created again my project, but the problem persist.

Can anyone suggest something? I would really appreciate that.

**Comment by freshyill** [comment, marked as answer]
Did you get this sorted out?

Here's how the plugin sets permalinks. If it finds the tag `gardenEntry`, then it sets the permalink to be the homepage.

```js
permalink: (data) => {
  if (data.tags.indexOf("gardenEntry") != -1) {
    return "/";
  }
  return data.permalink || undefined;
},
```
It looks like that gets added when you check the "dg-home" checkbox in the Obsidian plugin. Are you using `gardenEntry` or something similar as a tag on your notes? If so, try removing the tag.

**Reply by Njthon** [reply]
Hello,

yes you are exactly right, that was the problem. I was indicating the tag dg-home in all the pages. Thanks for the reply anyway :)
